cowprotocol/cow-amm
# Add tests for `supportsInterface`

`supportsInterface` is the only function for which we don't have full coverage. (And `defaultBalancerVault`, but this is fixed in another PR.)
I use coverage fairly often to make sure I'm not missing anything and that failed coverage is quite distracting.

### How to test

Only tests are added.
Try `forge coverage` and confirm that :
```
| src/ConstantProduct.sol  | 100.00% (60/60)  | 100.00% (74/74)  | 100.00% (26/26) | 100.00% (7/7)  |
```
